app/models/journeyDomain/routeDetails/locationOfGoods/LocationOfGoodsDomain.scala

```diff
@@ -84,16 +85,18 @@ object LocationOfGoodsDomain {
     def userAnswersReader(typeOfLocation: LocationType): UserAnswersReader[LocationOfGoodsDomain] =
       (
         UserAnswersReader(typeOfLocation),
-        LocationOfGoodsEoriPage.reader
+        LocationOfGoodsEoriPage.reader,
+        LocationOfGoodsAddIdentifierPage.reader
```

Comment:
For now this could be something like `LocationOfGoodsAddIdentifierPage.filterOptionalDependent(identity)(UserAnswersReader(""))`. This will read the identifier as an empty string until the page has been built and you can add the actual `{{page}}.reader` implementation